Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/RideRequestUtils.java

```diff
@@ -406,4 +417,53 @@ public static void updateDriverStatus(Boolean status, String mode, Context conte
             }
         });
     }
+
+    public static void addRideReceivedEvent(JSONObject entity_payload, Bundle rideRequestBundle, SheetModel model, String event, Context context) {
+        ExecutorManager.runOnBackgroundThread(() -> {
```

Comment:
is it okay to push log for every ride request to clevertap?

Reply:
Yes this is what we want